class-tgm-plugin-activation.php

```diff
@@ -2023,6 +2039,7 @@ public function force_activation() {
 					} elseif ( $this->can_plugin_activate( $slug ) ) {
 						// There we go, activate the plugin.
 						activate_plugin( $plugin['file_path'] );
+						$this->force_activated_plugins[] = $slug;
```

Comment:
I'm thinking a transient should be set with this information. The situation - a plugin being force (re-) activated - could come up during a bulk-install which uses frames which would prevent the message from becoming visible. Similarly, another program could with code try to silently deactivate a plugin.

So, what about setting a transient with the slugs of force-activated plugins whenever this occurs ? Then having an admin_notice action which checks if such a transient exists, that we're in the WP back-end and that the current user can activate activate plugins and if so, shows a one-time notice to that user about the plugin(s) which were force activated ?